packages/redux/src/settings/types/state.types.ts

```diff
@@ -1,18 +1,38 @@
-import type { BlackoutError, Configuration } from '@farfetch/blackout-client';
+import type {
+  AccountSetting,
+  BlackoutError,
+  Configuration,
+} from '@farfetch/blackout-client';
 import type { CombinedState } from 'redux';
 
+export type AccountSettingsHash = string;
```

Comment:
I would not create this type as it will not be very useful since it will always be a string.

Reply:
Removed it 👍